plugins/nf-amazon/build.gradle

```diff
@@ -46,6 +46,8 @@ dependencies {
     api ('com.amazonaws:aws-java-sdk-ecs:1.12.129')
     api ('com.amazonaws:aws-java-sdk-logs:1.12.129')
     api ('com.amazonaws:aws-java-sdk-codecommit:1.12.129')
+    api ('com.amazonaws:aws-java-sdk-core:1.12.129')
```

Comment:
```suggestion
```

Think sdk-core is implicitly added as a root dependency. could be safely removed